build.gradle

```diff
@@ -215,6 +215,11 @@ okbuck {
         allowAllVersions = [
                 "org.robolectric:android-all",
         ]
+        autoValueConfigurations = [
```

Comment:
why not re-use extra configs for these?

Reply:
extra config adds `ExtraDepCache` as suffix. also, I am using this configuration name (in hyphen case) as the name of the processor.